source/six_fox.f90

```diff
@@ -2771,55 +2771,56 @@ end subroutine clorda
 !           note: inversion method copied from subroutine postpr        *
 !-----------------------------------------------------------------------*
 subroutine invert_tas(fma_tas_inv,fma_tas)
```

Comment:
Note the comment -- this subroutine is copied from postpr, and should probably be shared somehow...